### ISSUE TITLE
Need revised .ini for Windows 10.0.17763.775

### ISSUE THREAD

**Issue by allisongsa** [stascorp/rdpwrap#898]
Need revised .ini for Windows 10.0.17763.775

**Comment by PosGuys** [comment]
I need the Same.  Please post a fix for same version Windows10.

**Comment by malicon** [comment]
Same issue for me after a Windows 10 update. The termsrv.dll version 10.0.17763.775 is said to be not supported.  Need new entries in the rdpwrap.ini file.

**Comment by vangorpy** [comment]
Yes, got the same issue as well 10.0.17763.775 not supported.

**Comment by OPADA-Eng** [comment]
Here is the correct ini file.
[rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

**Comment by malicon** [comment]
> 
> 
> Here is the correct ini file.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

The ini files have two sections per termsrv.dll version.  
First section is like
[10.0.17763.292]
LocalOnlyPatch.x86=1
LocalOnlyOffset.x86=AFAD4
... followed by many more lines.

The second section is like
[10.0.17763.292-SLInit]
bInitialized.x86 =CD798
bServerSku.x86 =CD79C
... followed by many more line.

For a termsrv.dll version to be supported, RDPWarp must find in the ini file two sections like above that match the running dll version.  That's why we request sections for version 10.0.17768.775. 
What you've posted has no sections for that dll version.  It cannot work...

**Comment by s0nic9** [comment]
rdpwrap ini version has nothing to do with Windows version, the version needed must match termsrv.dll version only. not updated since months.

**Comment by neymp** [comment]
> Here is the correct ini file.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

Thanks! Ok with 17763.805 Windows version and 17763.771 termsrv version.

**Comment by OPADA-Eng** [comment]
Welcome :)

**Comment by JohnB69** [comment]
> 
> 
> > Here is the correct ini file.
> > [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)
> 
> The ini files have two sections per termsrv.dll version.
> First section is like
> [10.0.17763.292]
> LocalOnlyPatch.x86=1
> LocalOnlyOffset.x86=AFAD4
> ... followed by many more lines.
> 
> The second section is like
> [10.0.17763.292-SLInit]
> bInitialized.x86 =CD798
> bServerSku.x86 =CD79C
> ... followed by many more line.
> 
> For a termsrv.dll version to be supported, RDPWarp must find in the ini file two sections like above that match the running dll version. That's why we request sections for version 10.0.17768.775.
> What you've posted has no sections for that dll version. It cannot work...

@ Malicon: do you have a working version meanwhile? I need an updated version voor .775 as well

**Comment by malicon** [comment]
**JohnB69** .  No I don't.  If I find one, I will post it here.

**Comment by KLEPTOROTH** [comment]
Did we get a version for 775 yet?

**Comment by bezik46** [comment]
Can you please understand that Windows .805 still uses termsrv.dll version 10.0.17763.771

**Comment by KLEPTOROTH** [comment]
> Can you please understand that Windows .805 still uses termsrv.dll version 10.0.17763.771

Yeah I realized that a few minutes later.  I was on version Windows 775.  I only look at this like once every couple months when it breaks so I forget the dll version differs.

Thanks!

**Comment by cjcharles0** [comment]
I assume, because we need a new ini for each new version of Remote Desktop, that every time Windows updates RDP my simultaneous users feature will break? Is that right?

**Comment by malicon** [comment]
@cjcharles0 you're right!  Though it seems to only happen on major Windows feature updates.

**Comment by cjcharles0** [comment]
Thanks!
Can we offer assistance to become contributors to this github project, that way I guess we can easily fix the 'update' button inside the app?

**Comment by allanwlz** [comment]
So 775 is still not supportted yet?

**Comment by estivalinp** [comment]
> Here is the correct ini file.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

Thank you! It also works in my win10 1809 ver 17763.805

**Comment by malicon** [comment]
After a few days, my 3 PCs have declared that the last update did not finished properly and after I have re-applied the update, the termsrv.dll version is no longer 775 it's now 771.  Thus the ini files posted by @OPADA-Eng @neymp @estivalinp all work (they are the same) since they have entries for 711.  Thanks to the three of you.  My Win10 version is 1809 os build 177763.805

**Comment by ghost** [comment]
> Here is the correct ini file.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

This works for me, thks!!!

**Comment by lxwulf** [comment]
> 
> 
> Here is the correct ini file.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/3702885/rdpwrap.zip)

even works for 10.0.17763.771

![image](https://user-images.githubusercontent.com/44492381/75628191-ef0da800-5bd6-11ea-9a11-4bdd964d73ce.png)

thank you so much!

how can i add the new offsets when a new update is out?

**Comment by affinityv** [comment]
This [ini](https://github.com/affinityv/INI-RDPWRAP/blob/master/rdpwrap.ini) file has all the sections that I am aware of to date.

Edit: I try to keep an eye out for new ones... let me know if any are missing and where I can source the INI file section(s) to add them.

**Comment by lxwulf** [comment]
thank you @affinityv i would update my system in a couple of days to 1904 and give you feedback if it works and also when it doesnt in the future - also very motivated to help in any way i can im a novice programmer but want to learn =D

**Comment by OPADA-Eng** [comment]
@affinityv   Thanks, it works on update Version **10.0.18363 Build 18363**

**Comment by lxwulf** [comment]
Yeah, very nice with ini file from @affinityv it even works with ver. 10.0.18362.657 Update 1909 from Windows thank you again!


![image](https://user-images.githubusercontent.com/44492381/76167005-ad937480-6163-11ea-93b2-13fe3e67c7fb.png)
